src/python/bot/startup/health_check_responder.py

```diff
@@ -46,8 +47,9 @@ def do_GET(self):  # pylint: disable=invalid-name
 
 def run_server():
   """Start a HTTP server to respond to the health checker."""
-  if utils.is_oss_fuzz():
-    # OSS-Fuzz's multiple instances per host model isn't supported yet.
+  if utils.is_oss_fuzz() or environment.is_android():
```

Comment:
This may be a little too broad, and capture cases (e.g. Cuttlefish) where there is a single host per "device". 

Would it be possible to have this be narrower here?

Reply:
Hi @oliverchang thanks for the response..

`environment.is_android()` returns the env variable set for `OS_OVERRIDE`.

In the case of cuttlefish this is set to `ANDROID_X86`:

from `_internal/system/environment.py`:

```
def is_android(plt=None):
  """Return True if we are on android platform."""
  return 'ANDROID' in (plt or platform())


def is_android_cuttlefish(plt=None):
  """Return True if we are on android cuttlefish platform."""
  return 'ANDROID_X86' in (plt or platform())


def is_android_kernel(plt=None):
  """Return True if we are on android kernel platform groups."""
  return 'ANDROID_KERNEL' in (plt or get_platform_group())
```

I believe this means only "Real" devices are detected by `is_android`.

Would changing `is_android()` to `is_android_real_device()` address this concern?

Reply:
the `in` operator would cause the `'ANDROID' in 'ANDROID_X86'` to match, so `is_android` would return True for cuttlefish devices too. 

A new `is_android_real_device()` which explicitly excludes cuttlefish SGTM.

Reply:
Done